app/views/projects/blob/new.html.haml

```diff
@@ -1,5 +1,11 @@
-- page_title "New File", @ref
-%h3.page-title New file
+%h3.page-title<
+  Create new file or 
+  %a.button{href: '#modal-upload-blob', 'data-target' => '#modal-upload-blob', 'data-toggle' => 'modal'}> &nbsp;upload existing one
```

Comment:
With the latest design change, the "upload existing one" isn't a different color:

![image](https://cloud.githubusercontent.com/assets/963826/9837795/a21824ea-59ff-11e5-93d5-322718c601f8.png)

Also, I think it looks better without the space highlighted:

![image](https://cloud.githubusercontent.com/assets/963826/9837803/bb36d84a-59ff-11e5-9ec3-ca809aec00fb.png)